### PR TITLE
Enable System.IO.FileSystem tests for native aot

### PR DESCRIPTION
**Pull request by jkotas**
Suppress file sharing tests on machines that do not have file sharing service installed.

### CODE REVIEW

**Comment by jkotas** [comment]
Native aot is apparently the only config that runs outer loop tests as admin on Win2022 Spanish Helix machines that do not have file sharing service installed.

File: src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.Windows.cs [hunk 1]
```diff
@@ -85,15 +85,24 @@ public class UncFilePathFileStreamStandaloneConformanceTests : UnbufferedAsyncFi
...
             }
 
-            // the "Server Service" allows for file sharing. It can be disabled on some of our CI machines.
-            using (ServiceController sharingService = new ServiceController("Server"))
+            try
+            {
+                // the "Server Service" allows for file sharing. It can be disabled on some machines.
+                using (ServiceController sharingService = new ServiceController("Server"))
+                {
+                    Console.WriteLine(sharingService.Status == ServiceControllerStatus.Running);
```
Comment (danmoseley) [inline src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.Windows.cs:98]: do you want to leave this console write in?

**Review by danmoseley — APPROVED** [review]

File: src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.Windows.cs (hunk 1 above)
Comment (jkotas) [inline src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.Windows.cs:98]:
```suggestion
```

**Pull request merged**